docs/getting-started/advanced-setting.asciidoc

```diff
@@ -71,9 +71,9 @@ collectors to ship data to {es}, the data must be mapped to ECS.
 [[update-threat-intel-indices]]
 == Update default Elastic Security threat intelligence indices
 
-The `securitySolution:defaultThreatIndex` advanced setting defines threat intelligence indices that {elastic-sec} will use when collecting threat indicators. The setting controls features that query threat indices, such as the Threat Intelligence view on the Overview page and threat indicator match rules. One or more threat intelligence indices can be defined; the `filebeat-*` index is specified by default.
+The `securitySolution:defaultThreatIndex` advanced setting defines threat intelligence indices that {elastic-sec} features use when querying for ingested threat indicators. The setting controls features that query threat intelligence indices, such as the Threat Intelligence view on the Overview page, indicator match rules, and the alert enrichment query. One or more threat intelligence indices can be defined; multiple indices must be separated by commas. By default, the `logs-ti*` index pattern is defined and used by {agent} integrations.
 
-IMPORTANT: Threat intelligence indices aren't required to be ECS compatible when used in an indicator match rule. However, we recommend compatibility if you'd like for your alerts to be enriched with relevant threat indicator information. By default, indicator match rules are configured to ingest indicators from the Threat Intel {filebeat} module and to check for fields prefixed with `threatintel.indicator`. See {filebeat-ref}/filebeat-module-threatintel.html[Threat Intel module] for more information about these fields. This rule setting is stored in the *Indicator prefix override* field, under indicator match rule advanced settings. If your indicator data contains indicator fields at a different location, update this field accordingly to ensure alert enrichment can still be performed.
+IMPORTANT: Threat intelligence indices aren't required to be ECS compatible when used in an indicator match rule. However, we strongly recommend compatibility if you'd like for your alerts to be enriched with relevant threat indicator information. When searching for threat indicator data, indicator match rules uses the threat indicator path specified in the *Indicator prefix override* advanced setting. Visit <<rule-ui-advanced-params, Configure advanced rule settings>> for more information.
```

Comment:
```suggestion
IMPORTANT: Threat intelligence indices aren't required to be ECS compatible for use in indicator match rules. However, we strongly recommend compatibility if you want your alerts to be enriched with relevant threat indicator information. When searching for threat indicator data, indicator match rules use the threat indicator path specified in the *Indicator prefix override* advanced setting. Visit <<rule-ui-advanced-params, Configure advanced rule settings>> for more information.
```

Reply:
+1 to @benironside's edit, and also hyphenate "ECS-compatible"